testing/functional/tests/scheduler/dragAndDrop/day.ts

```diff
@@ -0,0 +1,107 @@
+import { ClientFunction } from 'testcafe';
+
+import { getContainerFileUrl } from '../../../helpers/testHelper';
+
+import { dataSource } from './init/widget.data';
+import { createScheduler } from './init/widget.setup';
+
+import { AppointmentModel, Property } from './helpers/appointment.helper';
+
+import { TablePosition } from './helpers/props/tablePosition';
+import { Size } from './helpers/props/size';
+import { TimeSpan } from './helpers/props/timeSpan';
+
+import { dropTo, compare } from './helpers/actions.helper';
+
+fixture`Rearrange appointments in the Scheduler widget with the drag-and-drop gesture`
+    .page(getContainerFileUrl());
+
+test('Drag-and-drop appointments in day', async t => {
+    let appointment;
+    const comparsionProperties = [
+        Property.height,
+        Property.startTime,
+        Property.endTime
+    ];
+
+    appointment = new AppointmentModel('Appointment #1',
+        {
+            position: new TablePosition(1, 0),
+            size: new Size('auto', '50px'),
+            duration: new TimeSpan('9:30 AM', '10:00 AM')
+        });
+
+    await dropTo(t, appointment.title, appointment.properties.position);
+    await compare(t, appointment.title, appointment.properties, comparsionProperties);
+
+    appointment = new AppointmentModel('Appointment #1',
+        {
+            position: new TablePosition(3, 0),
+            size: new Size('auto', '50px'),
+            duration: new TimeSpan('10:30 AM', '11:00 AM')
+        });
+
+    await dropTo(t, appointment.title, appointment.properties.position);
+    await compare(t, appointment.title, appointment.properties, comparsionProperties);
+
+    appointment = new AppointmentModel('Appointment #2',
+        {
+            position: new TablePosition(1, 0),
+            size: new Size('auto', '100px'),
+            duration: new TimeSpan('9:30 AM', '10:30 AM')
+        });
+
+    await dropTo(t, appointment.title, appointment.properties.position);
+    await compare(t, appointment.title, appointment.properties, comparsionProperties);
+
+    appointment = new AppointmentModel('Appointment #2',
```

Comment:
Remove duplicate Draggings (they are similar to each other) or add comments in code explaining why they are needed